core/src/main/resources/jenkins/console/ConsoleUrlProviderUserProperty/config.properties

```diff
@@ -0,0 +1 @@
+description=Controls what visualization tool handles build console links. Providers higher in the list take precedence over providers lower in the list.
```

Comment:
Ideally this would explain the relationship to the global configuration by the admin (precedence / override).

Reply:
Let me know what you think about [64c3faa](https://github.com/jenkinsci/jenkins/pull/8321/commits/64c3faa8ac16742878377db5f9b0d14ff95a1338).